include/podio/ROOTReader.h

```diff
@@ -26,6 +25,8 @@ class CollectionBase;
 class Registry;
 class CollectionIDTable;
 class GenericParameters;
+struct CollectionBuffers;
+
```

Comment:
This does no longer exist? Should probably be `CollectionReadBuffers`. Is the forward declaration necessary?